pkgs/top-level/all-packages.nix

```diff
@@ -5581,6 +5581,20 @@ in
 
   srcml = callPackage ../applications/version-management/srcml { };
 
+  srhtPackages = callPackage ../applications/version-management/sr-ht {
+    # https://github.com/NixOS/nixpkgs/pull/54425#discussion_r249489825
+    pythonPackages = python36Packages;
+    fetchNodeModules = callPackage ../applications/networking/instant-messengers/rambox/fetchNodeModules.nix { };
```

Comment:
This can also be put in a let statement inside the expression. However, I'm not sure if `fetchNodeModules` should be used, I'm not familiar with the Node ecosystem.